docs/attributes-registry/server.md

```diff
@@ -1,23 +1,18 @@
 <!--- Hugo front matter used to generate the website version of this page:
-linkTitle: Server
 --->
 
-# Server Attributes
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
 
-These attributes may be used to describe the server in a connection-based network interaction
-where there is one side that initiates the connection (the client is the side that initiates the connection).
-This covers all TCP network interactions since TCP is connection-based and one side initiates the
-connection (an exception is made for peer-to-peer communication over TCP where the "user-facing" surface of the
-protocol / API does not expose a clear notion of client and server).
-This also covers UDP network interactions where one side initiates the interaction, e.g. QUIC (HTTP/3) and DNS.
```

Comment:
Can this maybe be included in the attribute group notes?

Reply:
If we render those, yes. I'll look into doing that shortly.